lib/uo_cli.rb

```diff
@@ -68,7 +70,7 @@ class UOCLI
     # params\
     # +scenario+:: _string_ Name of csv file that defines the scenario\
     # +featureFile+:: _string_ Name of Feature File used to describe set of features in the district. If not passed, uses example project.
-    def scenario(scenario, featureFile=nil)
+    def run_func(scenario, featureFile=nil)
       featureFile = "example_project.json" if featureFile.nil?
       name = "#{scenario.capitalize} Scenario"
       root_dir = File.dirname(__FILE__)
```

Comment:
these paths made sense when called from the rakefile in the example project, but now that the CLI has been broken out, we need to fix these.  I think what makes the most sense is to pass in the path to scenario CSV and featurefile instead of just the filename.

Reply:
scenario and featurefile now need to be called with absolute paths